iothub/service/tests/SerializationTests.cs

```diff
@@ -24,7 +25,8 @@ public void Twin_JsonDateParse_Ok()
  ""lastActivityTime"": ""2018-06-29T21:17:08.7759733"",
 }";
 
-            JsonConvert.DeserializeObject<ClientTwin>(jsonString);
+            ClientTwin clientTwin = JsonConvert.DeserializeObject<ClientTwin>(jsonString);
+            clientTwin.DeviceId.Should().Be("test");
```

Comment:
If you can, make a constant for this so we don't have to rely on static strings matching.